bcda/auth/client/oktaclient.go

```diff
@@ -149,6 +164,54 @@ func (oc *OktaClient) AddClientApplication(localID string) (string, string, erro
 	return clientID, clientSecret, nil
 }
 
+func (oc OktaClient) RequestAccessToken(creds Credentials) (OktaToken, error) {
```

Comment:
other funcs use the pointer to OktaClient as the receiver.  should we do that here as well?